contracts/vortex/CarbonVortex.sol

```diff
@@ -1012,6 +1032,19 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
         return price;
     }
 
+    function _transferProceeds(Token token, uint256 amount) private {
+        // if transfer address is 0, proceeds stay in the vortex
+        if (_transferAddress == address(0)) {
+            // increment totalCollected amount
+            _totalCollected += amount;
```

Comment:
Nitpick: you can call this first and then you can call it once (CEI isn't essential here, but it's your call)

Reply:
fixed, was thinking about it as well